san-francisco-tourism/build.gradle

```diff
@@ -0,0 +1,33 @@
+
+group = 'com.oracle.javaone.2017.hol.junit5'
+version = '0.0.1-SNAPSHOT'
+
+description = "junit5 with gradle as build system"
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+ext.junitVintageVersion  = '5.0.0'
+ext.junitPlatformVersion = '1.2.0'
+ext.junitVersion  = '5.0.0'
+
+
+repositories {
+        
+     maven { url "http://repo.maven.apache.org/maven2" }
+}
+dependencies {
+    compile group: 'org.mockito', name: 'mockito-core', version:'2.9.0'
```

Comment:
Can you be consistent with variables, either specify them all inline or use variables (**i prefer variables**)
e.g. _ext.version = foo_